src/library_webaudio.js

```diff
@@ -249,14 +249,15 @@ let LibraryWebAudio = {
 #endif
 
     EmAudio[contextHandle].audioWorklet.bootstrapMessage.port.postMessage({
-      // '_wpn' == 'Worklet Processor Name', use a deliberately mangled name so
-      // that this field won't accidentally be mixed with user submitted
-      // messages.
-      _wpn: UTF8ToString(HEAPU32[options]),
-      audioParams,
-      contextHandle,
-      callback,
-      userData
+      // Deliberately mangled and short names used here ('_wpn', the 'Worklet
+      // Processor Name', to not get accidentally mixed with user submitted
+      // messages, the remainder for space saving reasons, abbreviated from
+      // their variable names).
+      '_wpn': UTF8ToString(HEAPU32[options]),
+      'ap': audioParams,
+      'ch': contextHandle,
+      'cb': callback,
+      'ud': userData
```

Comment:
Makes me a bit sad that we have to do this in the source code rather than have the minifier tool do it... but I guess we don't have a better way.

Do we have any codesize test for audioworklets that can means that code size improvement here?  If not maybe we should add one.


I wonder if we should do this in the pthreads code too?  @juj what do you think of this technique?

Reply:
This looks fine to me.

In general end users should never interact/snoop these postMessage()s, so their format should be strictly internal to the implementation. The only thing here is to make sure we generate messages that are not easy to confuse with user-sent messages - which I think manually mangled names make pretty clear it won't be anything that a user might have written.

Reply:
So, IIUC, there is no difference between these mangled names and the existing `_wsc` and `_wpn` mangled names.   Can we be consistent and use underscores either for all of them, or none?

Reply:
One difference was that the `_wsc` and `_wpn` fields were the "keys" of the data object, used to identify what the postMessaged struct was. These other fields are the satellite data, that are assumed to be present when that one key field matches. That is why I initially only mangled and underscored that one field to make it clear that "that's the key, others are satellite" disposition.

Reply:
Fair enough, lgtm

Reply:
Actually I'm still a little confused.  It looks like `_wsc` and `_wpn` are both fields the same object that `ap` and `ch` are part of.   

For example I see ` registerProcessor(d['_wpn'], createWasmAudioWorkletProcessor(d['ap']));` where they are both fields of `d`.

Reply:
I think the importance of `_wpn` is that no other message is going to have such a property. There's a test, equivalent to
```
if (msg.data['_wpn']) {
```
I can imagine minified properties being called `ap`, which might trigger a false positive, but it's unlikely there'd be a `_wpn`.

So my take, I can see reasoning for the underscore here, but for anything else it's just extra characters.